finance/sc-im/Portfile

```diff
@@ -0,0 +1,27 @@
+PortSystem          1.0
```

Comment:
can you please add our default modeline as the first line of the Portfile?
```# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4```

Reply:
to make `port lint` happy there please add a newline after the `modeline` and after the line with `PortSystem`